core/spec.md

```diff
@@ -6143,107 +6211,121 @@ those cases, the client will need to query the individual inlinable attributes
 in isolation so the Registry can leverage
 [pagination](../pagination/spec.md) of the response.
 
-#### Inline
+#### Collections Flag
 
-The `?inline` query parameter (flag) MAY be used on requests to indicate whether
-nested collections/objects, or certain (potentially large) attributes, are to
-be included in the response message.
-
-The `?inline` query parameter on a request indicates that the response
-MUST include the contents of all specified inlinable attributes. Inlinable
-attributes include:
-- The `model` attribute on the Registry entity.
-- The `capabilities` attribute on the Registry entity.
-- All [Registry Collection](#registry-collections) types - e.g. `GROUPS`,
-  `RESOURCES` and `versions`.
-- The `RESOURCE` attribute in a Resource or Version.
-- The `meta` attribute in a Resource.
+The `?collections` query parameter (flag) MAY be used on requests directed
+to the Registry itself or to Group instance to indicate that the response
+message MUST NOT include any attributes from the top-level entity (Registry
+or Group), but instead MUST include only all of the nested Collection maps
+that are defined at that level.
 
-While the `RESOURCE` and `RESOURCEbase64` attributes are defined as two
-separate attributes, they are technically two separate "views" of the same
-underlying data. As such, the usage of each will be based on the content type
-of the Resource, specifying `RESOURCE` in the `?inline` query parameter MUST
-be interpreted as a request for the appropriate attribute. In other words,
-`RESOURCEbase64` is not a valid inlinable attribute name.
+Note that this feature only applies to the root entity of the response and not
+to any nested entities/collections.
 
-Use of this feature is useful for cases where the contents of the Registry are
-to be represented as a single (self-contained) document.
+This feature is meant to be used when the Collections of the Registry, or
+Group, are of interest but not the top-level metadata. For example, to export
```

Comment:
```suggestion
Group, are of interest but not the top-level metadata. For example, this could be used to export
```

(Or something similar - otherwise it doesn't sound like a complete sentence to me.)

Reply:
done